frontend/packages/dev-console/src/components/topology-list/TopologyToggleIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { chart_global_Fill_Color_900 as Color900 } from '@patternfly/react-tokens';
```

Comment:
This doesn't feel right - do we not have a better mechanism for applying colours?

cc @christianvogt @rohitkrai03